.gitlab-ci.yml

```diff
@@ -0,0 +1,19 @@
+# Upload everything to staging
+ stages:
+   - test
+   - deploy
+ 
+ 
+ deploy to data.trezor.io:
+  stage: deploy
+  needs: []
+  before_script: []
+  only:
+    refs:
+      - master
+  when: manual
+  script:
+    - source ${DEPLOY_DATA_KEYFILE}
+    - ./s3sync.sh
```

Comment:
Won't [this](https://github.com/trezor/webwallet-data/blob/master/s3sync.sh#L21) cause a problem?

Reply:
But there it is called only for production, https://github.com/trezor/trezor-suite/blob/develop/packages/suite-web/scripts/s3sync.sh#L43.

I suggest to:
- Create a CI dir and move the s3sync script there.
- Remove the confirmation.

And btw it would be nice to do the same in trezor-suite (move the script to ci folder).

Reply:
changed it